airflow-core/src/airflow/utils/log/file_task_handler.py

```diff
@@ -407,7 +425,7 @@ def _read(
         logs = list(
             _interleave_logs(
                 *local_logs,
-                *remote_logs,
+                *(remote_logs or []),
```

Comment:
Do we need this since we have a default set?

Reply:
Mypy says yes (the return value could be none I think?